services/explorer/src/components/TransactionRow/TransactionRow.tsx

```diff
@@ -1,13 +1,14 @@
 import { Icons, Label } from '@taraxa_project/taraxa-ui';
 import moment from 'moment';
 import React from 'react';
+import { BlockData } from '../../pages/Blocks/Blocks.effects';
```

Comment:
Ideally we should not import files from effects in components. If we use the files here, then we should have the BlockData somewhere outside, in a models folder. Same goes for the imports from Transaction.effects. Let's move these models outside (in a models folder, table.ts file) and then import them here and in the effects as well.